.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: minimal
+services:
+- docker
+script: cd infrastructure-tests && chmod +x runTests.sh && ./runTests.sh
+env:
+  global:
+  - PPIPER_INFRA_IT_TEST_PROJECT=https://github.com/SAP/cloud-s4-sdk-book
+  - PPIPER_INFRA_IT_CF_USERNAME=P2001099548
```

Comment:
Not sure if the P-User should be part of this repo, should be configured together with the password in travis

Reply:
This is the configuration in travis. The password is also in this file (encrypted). Not sure if travis provides another way to configure this.

Reply:
I can also configure it in the UI of Travis ...